### DISCUSSION TITLE
What is another name for the °C 🌡️ scale?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#920] [Galaxy Trivia] [answered]
**Bonus:** Convert 40°C to the following scales: K, °F, °Ra, °Rø, °N, °D, °R.

**Comment by ParanoidUser** [comment]
@Marcoant007 / @Querela, up for the challenge?

**Reply by Querela** [reply]
Nope

**Comment by peterbaccaro** [comment, marked as answer]
Other names for the °C temperature scale are **Celsius scale** and **Centigrade scale**.

Centigrade is the old fashioned name for Celsius. The name Centigrade was derived from the Latin originally meaning a hundred degrees. The original scale was created by Anders Celsius in 1742.

40°C is equivalent to:

- 40°C = 40 + 273.15 = **313.15 Kelvin (K)**
- 40°C = (40 × 9/5) + 32 = **104°Fahrenheit (°F)**
- 40°C = (40 × 9/5) + 491.67 = **563.67 Rankine (°Ra)**
- 40°C = (40 × 21/40) + 7.5 = **30.5 Rømer (°Rø)**
- 40°C = 40 × 0.33 = **13.2 Newton (°N)**
- 40°C = (100 - 40) × 3/2 = **30 Delisle (°D)**
- 40°C = 40 × 4/5 = **32 Réaumur (°R)**
